src/components/data_comps_nuopc/datm/src/atm_comp_nuopc.F90

```diff
@@ -1094,6 +1096,10 @@ subroutine datm_comp_realize(importState, exportState, rc)
     call dshr_dfield_add( dfields, sdat, state_fld='Faxa_lwdn' , strm_fld='lwdn', &
          state=exportState, state_ptr=Faxa_lwdn, strm_ptr=strm_lwdn, logunit=logunit, masterproc=masterproc, rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    call dshr_dfield_add( dfields, sdat, state_fld='Faxa_swdn'  , strm_fld='swdn'  , &
```

Comment:
See above comment.

Reply:
The Faxa_swdn is also added to esmFldsExchange_cesm_mod.F90 to make A compset run.